packages/mistral/src/mistral-chat-options.ts

```diff
@@ -8,6 +8,9 @@ export type MistralChatModelId =
   | 'mistral-large-latest'
   | 'mistral-small-latest'
   | 'pixtral-large-latest'
+  // reasoning models
+  | 'magistral-small-2506'
+  | 'magistral-medium-2506'
```

Comment:
also needs docs updates, the models are listed on the mistral provider page and potentially some index pages